src/link.zig

```diff
@@ -282,6 +283,16 @@ pub const HashStyle = enum { sysv, gnu, both };
 
 pub const CompressDebugSections = enum { none, zlib };
 
+/// The filesystem layout of darwin SDK elements.
+pub const DarwinSdkLayout = enum {
+    /// Does not apply to the target.
+    none,
```

Comment:
I'm thinking that if that is the case, perhaps we should reduce this further into `?{ .sdk, .vendored }` where `null` would signal that it's not applicable?

Reply:
Indeed, I think that adds clarity.